sample-code/pmd-example-rules/README.md

```diff
@@ -0,0 +1,17 @@
+# PMD example rules
+Salesforce Code Analyzer supports the use of custom PMD rules. If those rules involve any Java code,
```

Comment:
PMD Example Rules

Salesforce Code Analyzer supports the use of custom PMD rules. If your custom PMD rules include Java code, then you must compile the Java code into a JAR before you register it with Code Analyzer.

This project is an example of how to write Java-based PMD rules, and use Maven to compile them into a JAR.

Reply:
@jfeingold35 See edits in the G doc: https://docs.google.com/document/d/1YAOmc63r1y95SD8N-8Ty0n13uPDfNR89rV8ByXLEV0M/edit?usp=sharing